### PR TITLE
add AI warning on create page load

### PR DESCRIPTION
**Pull request by callumflack**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const config = {
-  VANA_API_URL: 'https://api.vana.com/api/v0',
-  VANA_GITHUB_URL: 'https://github.com/corsali/vana-portrait-demo',
-  VANA_PORTRAIT_URL: 'https://portrait.vana.com',
+  VANA_API_URL: "https://api.vana.com/api/v0",
+  VANA_GITHUB_URL: "https://github.com/corsali/vana-portrait-demo",
+  VANA_PORTRAIT_URL: "https://portrait.vana.com",
+  vanaSupportEmail: "support@vana.com",
```
Comment (dmitryvlass) [inline config.js:5]: nit: could you follow the UPPER_CASE format?

Comment (dmitryvlass) [inline config.js:5]:
```suggestion
  VANA_SUPPORT_EMAIL: "support@vana.com",
```

**Review by dmitryvlass — APPROVED** [review]
Code LGTM

File: config.js (hunk 1 above)
Comment (callumflack) [reply config.js]: Yep, thanks!

**Pull request merged**